packages/next/src/build/utils.ts

```diff
@@ -1676,7 +1687,7 @@ export async function copyTracedFiles(
   const serverOutputPath = path.join(
     outputPath,
     path.relative(tracingRoot, dir),
-    'server.js'
+    moduleType ? 'server.mjs' : 'server.js'
```

Comment:
shouldn't we output `server.cjs` since a lot of things are required to run under cjs now. for instance the precompiled modules are all cjs. if we have `require(module)` it will not work well under mjs?

Reply:
oh it's already outputing esm content. but `.js` is acting same as `.mjs`. why we need to specify this extension 🤔

Reply:
We could always keep it as `.js` potentially although it's less clear we are explicitly creating a `.mjs` server file.

Reply:
I'm gonna revert the extension change to avoid un-necessary thrash as copying the package.json always should be sufficient